### DISCUSSION TITLE
Form with discount

### DISCUSSION THREAD

**Discussion by LuSome** [liedekef/events-made-easy#489] [Q&A]
Is there an example how to make a form  where you can add a checkbox to receive a discount when checked.

**Comment by liedekef** [comment]
Sure: create a checkbox field and set "extra charge" to "yes", the field value to the (negative) price to be used and the field value to the "visible" text people get to see.
![image](https://github.com/liedekef/events-made-easy/assets/75667/9726bf4a-02ee-433e-b64e-54e273344802)
![image](https://github.com/liedekef/events-made-easy/assets/75667/30ea4f19-8d67-461e-8c80-f561e5cff73b)

**Reply by LuSome** [reply]
Thanks, this works for me.  But how can you use Discounts from the program and use them in a registration form?

**Reply by liedekef** [reply]
That was not your original question. To use discounts: define your discounts and or discount groups, add them to the event definition and then in the RSVP form use #_DISCOUNT where you want people to enter the discount code if appropriate. If #_DISCOUNT is not present, EME will add it dynamically at the end of the form (which is normally not what you want).

**Reply by LuSome** [reply]
Thanks.  I use it in https://boost-sportkampen.be/
